lib/k8s-e2e-tests/src/lib.rs

```diff
@@ -1,32 +1,69 @@
+use indoc::indoc;
 use k8s_openapi::{
-    api::core::v1::{Container, Pod, PodSpec},
-    apimachinery::pkg::apis::meta::v1::ObjectMeta,
+    api::core::v1::{Affinity, Container, Pod, PodAffinity, PodAffinityTerm, PodSpec},
+    apimachinery::pkg::apis::meta::v1::{LabelSelector, ObjectMeta},
 };
-use k8s_test_framework::{Framework, Interface, Reader};
+use k8s_test_framework::{
+    test_pod, wait_for_resource::WaitFor, CommandBuilder, Framework, Interface, Manager, Reader,
+};
+use std::collections::BTreeMap;
 use std::env;
+use tracing::{debug, error, info};
 
 pub mod metrics;
 
 pub const BUSYBOX_IMAGE: &str = "busybox:1.28";
 
+pub fn init() {
+    let _ = env_logger::builder().is_test(true).try_init();
+}
```

Comment:
Assuming `try_init` returns a result, maybe `unwrap` to exit if this fails?

Reply:
We do need to ignore the error here since init will fall over if it is called twice. The second test doesn't need this call and will error. It does feel like a bit of a hack, but it's recommended in the [docs](https://docs.rs/env_logger/0.8.3/env_logger/#capturing-logs-in-tests) to take this approach.